### DISCUSSION TITLE
Why does rollup bundle devDependencies instead of dependencies?

### DISCUSSION THREAD

**Discussion by steve-taylor** [sveltejs/kit#10908] [Q&A]
Pretty much what the title says. Why would things like `typescript`, `eslint`, `vitest`, etc., all of which should be `devDependencies`, be considered when bundling the app? Shouldn't rollup instead bundle packages that are in `dependencies`?

As it stands right now, in order to have some sort of deployable app, I have to swap my `dependencies` and `devDependencies` in my `package.json` which is just weird.

**Comment by teemingc** [comment]
It’s hard to say without a reproduction to look at. ~Also, SvelteKit doesn’t use rollup.~ Maybe it would be more helpful to ask in the rollup discussions?

EDIT: SvelteKit uses Vite which uses Rollup
